atomicapp/utils.py

```diff
@@ -354,6 +354,10 @@ def getRoot():
         else:
             return "/"
 
+    @staticmethod
+    def get_abspath(path):
+        return os.path.join(Utils.getRoot(), path)
```

Comment:
I think that we should add `.lstrip('/')` to `path` here, as it should be always called with `lstrip('/')`

`getRoot` returns '/host' or '/'
with leading '/' in second argument `os.path.join` cuts first part in path
```
>>> os.path.join("/host","/foo/bar")
'/foo/bar'

>>> os.path.join("/host","foo/bar")
'/host/foo/bar'
```